src/main/java/ChatReques21.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatReques21 {
+    private String model;
+    private List<?> messages;
```

Comment:
:information_source: Using wildcard type List<?> is not recommended. Specify the type explicitly. (Severity: Minor)